algorithms/src/snark/marlin/ahp/ahp.rs

```diff
@@ -47,7 +47,7 @@ pub(crate) fn witness_label(poly: &str, i: usize) -> String {
 impl<F: PrimeField, MM: MarlinMode> AHPForR1CS<F, MM> {
     /// The linear combinations that are statically known to evaluate to zero.
     #[rustfmt::skip]
-    pub const LC_WITH_ZERO_EVAL: [&'static str; 2] = ["matrix_sumcheck", "lincheck_sumcheck"];
```

Comment:
Let's switch this the other way: update the documentation to match these more informative names.

Reply:
Reverted